docs/source/modin/supported/general_supported.rst

```diff
@@ -64,6 +64,12 @@ Data manipulations
 |                             |                                 |                                  | performance. You can force the use of the Snowflake|
 |                             |                                 |                                  | parser with ``engine=snowflake``                   |
 +-----------------------------+---------------------------------+----------------------------------+----------------------------------------------------+
+| ``read_excel``              | D                               |                                  | Uses local pandas to read excel files, using the   | 
```

Comment:
nit: I always got some push-back on using the phrase "local". There seems to be a preference at Snowflake to use "client" instead. If it is not too much to ask, maybe would recommend reformulating here a bit. I also think 

>> Please refer to the native pandas documentation for more details.

could be either 1.) replaced with a link or 2.) left out when we throw a good error message.